scripts/faster_execution.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# This script greatly speeds up execution of qutebrowser commands when a
+# qutebrowser instance already exists.  Call it in place of your system's
+# qutebrowser executable.  It is based on the following work:
+#
+# scripts/open_url_in_instance.sh
+
+import codecs
+import hashlib
+import json
+import os
+import os.path
+import socket
+import subprocess
+import sys
+
+version = '1.0.4'
```

Comment:
While you're at it, you can probably update that to 1.6.0.

Reply:
Done.